Document `useWaitlist()` (#2754)
clerkinc/clerk-docs: docs/reference/javascript/waitlist.mdx

---
title: '`Waitlist`'
description: The `Waitlist` object holds the state of a waitlist entry and provides methods to join the waitlist.
sdk: js-frontend
---

> The `Waitlist` object provides methods and properties to manage waitlist entries in your application. It is used when you want to build a custom waitlist UI instead of using the prebuilt [`<Waitlist />`](/docs/reference/components/authentication/waitlist) component.

Reviewer: ```suggestion
The `Waitlist` object provides methods and properties to manage waitlist entries in your application. It's useful for building a [custom flow](!custom-flow) instead of using the prebuilt [`<Waitlist />`](/docs/reference/components/authentication/waitlist) component.
```
